### PR TITLE
Update Popovers

### PR DESCRIPTION
**Pull request by bestguy**
- Expose isOpen vs debugShowMode, useful as controlled prop
- Add support for hover and focus triggers
- Update stories

### CODE REVIEW

**Comment by bestguy** [comment]
Hi @eddie0329 , I've made some updates to Popover and props, if you'd like to review.
새해 복 많이 받으세요

File: src/Popover.svelte [hunk 1]
```diff
@@ -7,11 +7,11 @@
   export { className as class };
   export let children = undefined;
   export let animation = true;
+  export let hover = false;
   export let placement = 'top';
   export let target = '';
   export let title = '';
-  export let debugShowMode = false;
-  let isPopoverShow = false;
+  export let isOpen = false;
```
Comment (eddie0329) [inline src/Popover.svelte:14]: Wow... intergerating props value! This looks awesome !

File: stories/popover/Controlled.svelte [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <Popover
+    bind:isOpen
+    placement="right"
+    target="controlledBtn"
+    title="Controlled Popover">
+    This is a Popover controlled externally. 
+  </Popover>
+  <hr />
+  <label>
+    <input type="checkbox" bind:checked={isOpen} />
```
Comment (eddie0329) [inline stories/popover/Controlled.svelte:19]: Expose isOpen really makes user to use popover more extensively !

**Review by eddie0329 — APPROVED** [review]
Thank you @bestguy for give me opportunity to look over what's changed.
And i learned alot. I really appreciate. 🎉  Hope there is more work left for sveltestrap that i can involved more !
새해 복 많이 받으세요 too !

**Pull request merged**